.travis/smoke-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+function finish {
+    docker-compose --file docker-compose.yml --file docker-compose.ci.yml logs
+    docker-compose --file docker-compose.yml --file docker-compose.ci.yml down --volumes
+}
+
+trap finish EXIT
+
+docker-compose --file docker-compose.yml --file docker-compose.ci.yml up -d
```

Comment:
Probably don't want CI, and limit it to `ui` (not that it does any limiting at the moment).

Should add a healthcheck and use `wait-healthy.sh` too. Means we can lose https://github.com/libero/sample-configuration/blob/cabdcd93d45a04402c07bfbf5d460dac3162b8d7/docker-compose.yml#L20-L22.